src/components/search/advancedSearch/advancedSearch/AdvancedSearch.module.css

```diff
@@ -50,18 +50,31 @@
   display: flex;
 }
 
+/*
+NOTES
+463- Brug mobil-layout
+464+ Tekst til “Ryd” og “Hjælp” skal bruge desktopversionen.
+640+ Index-dropdowns og søgefelt skal ligge på samme linje.
+768+ Avanceret søgning + links + luk placeres som på desktop
+992+ Brug desktop-layout med ekstra (tom) spalte i venstre side og synlig topbar med simpleSearch, log ind etc.
+ */
+/* 640 and down */
 @media (max-width: 768px) {
   .buttonContainer {
     justify-content: start;
   }
 
   .closeContainer {
     position: absolute;
+    right: var(--pt2);
+    top: var(--pt3);
   }
+}
 
-  .closeContainer i {
-    position: relative;
-    top: 3px;
+/* 330px down (very small devices) */
+@media (max-width: 330px) {
```

Comment:
Alle andre steder i vores kode bruger vi `min-width` i stedet for `max-width`. Jeg ville nok foretrække at vi er konsekvente.